### PR TITLE
Add Giza council 43 reports

### PR DESCRIPTION
**Pull request by Ilichhh**

### CODE REVIEW

File: council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Negative value indicates deflation
 
 | Property                | Start Block | End Block | % Change |
 |-------------------------|--------------|--------------|----------|
-| Number of  Workers | 29 | 29 | 0 |
+| Number of  Workers | 9 | 8 | 0 |
```
Comment (traumschule) [inline council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md:99]: with [c5b328696185f8bc0b4c13ec78372337896d7405](https://github.com/Joystream/community-repo/pull/601/commits/c5b328696185f8bc0b4c13ec78372337896d7405)
> ### 4.2 Curators
> | Number of  Workers | 9 | 8 | -11.11 |

File: council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ Negative value indicates deflation
 
 | Property                | Start Block | End Block | % Change |
 |-------------------------|--------------|--------------|----------|
-| Number of  Workers | 58 | 58 | 0 |
+| Number of  Workers | 30 | 24 | 0 |
```
Comment (traumschule) [inline council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md:165]: with [c5b328696185f8bc0b4c13ec78372337896d7405](https://github.com/Joystream/community-repo/pull/601/commits/c5b328696185f8bc0b4c13ec78372337896d7405)
> ### 4.5 Operations
> | Number of  Workers | 30 | 24 | -20 |

File: council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+| 21 | [@seainvestor](https://pioneer.joystreamstats.live/#/members/seainvestor) | active | 3 | 0.5 | 16.8 |
+| 22 | [@0x2bc](https://pioneer.joystreamstats.live/#/members/0x2bc) | active | 43 | 1.8 | 37.1 |
+
+
+### 4.4 Distribution
+* [openings for distributionWorkingGroup](https://testnet.joystream.org/#/working-groups/opportunities/distribution)
+
+| Property                | Start Block | End Block | % Change |
+|-------------------------|--------------|--------------|----------|
+| Number of  Workers | 11 | 14 | 27.27 |
```
Comment (traumschule) [inline council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md:141]: with [c5b328696185f8bc0b4c13ec78372337896d7405](https://github.com/Joystream/community-repo/pull/601/commits/c5b328696185f8bc0b4c13ec78372337896d7405)
> ### 4.4 Distribution
> | Number of  Workers | 10 | 13 | 30 |

**Review by traumschule — CHANGES_REQUESTED** [review]
nice [find](https://discord.com/channels/811216481340751934/812343711870091285/944970047598764033) @mochet

**Comment by igrexac** [comment]
[Proposal](https://testnet.joystream.org/#/proposals/1182) approved

**Review by traumschule — APPROVED** [review]
LGTM!

**Review by freakstatic — APPROVED** [review]
LGTM!

**Review by freakstatic — APPROVED** [review]

File: council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+| 0 | [@leet_joy](https://pioneer.joystreamstats.live/#/members/leet_joy) | active | 87 | 2.2 | 2.2 |
+| 1 | [@igrex](https://pioneer.joystreamstats.live/#/members/igrex) | active | 4 | 0.1 | 0.1 |
+| 2 | [@marat_mu](https://pioneer.joystreamstats.live/#/members/marat_mu) | active | 4 | 0.1 | 0.1 |
+| 3 | [@kate_fm](https://pioneer.joystreamstats.live/#/members/kate_fm) | active | 4 | 0.1 | 0.1 |
+| 4 | [@adovrn](https://pioneer.joystreamstats.live/#/members/adovrn) | active | 0 | 0.0 | 0.0 |
+| 5 | [@kriptos](https://pioneer.joystreamstats.live/#/members/kriptos) | active | 0 | 0.0 | 0.0 |
+| 6 | [@shtihmas](https://pioneer.joystreamstats.live/#/members/shtihmas) | active | 0 | 0.0 | 0.0 |
+
+
+### 4.7 Gamma
```
[the remark this answers is not in the document]
Comment (traumschule) [reply council/tokenomics/giza/Council_Round43_4395600-4496399_Tokenomics_Report.md:219]: ok

**Pull request merged**